charts/kad/plugin_config.json

```diff
@@ -0,0 +1,13 @@
+{

```

Comment:
keep separate json file tekton and crossplane
configuration will be different, so it can't be loaded with struct in go.

in configmap, keep separate key for each plugin.

tekton_conifg: <data>
crossplane_conifg: <data>

refer https://github.com/intelops/vault-cred/blob/main/charts/vault-cred/templates/configmap_policies.yaml